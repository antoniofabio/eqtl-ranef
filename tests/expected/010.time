	Command being timed: "./test-010.sh"
	User time (seconds): 3.06
	System time (seconds): 0.08
	Percent of CPU this job got: 99%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:03.17
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 497568
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 36821
	Voluntary context switches: 23
	Involuntary context switches: 281
	Swaps: 0
	File system inputs: 0
	File system outputs: 16
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
