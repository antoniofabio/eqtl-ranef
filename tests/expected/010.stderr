List of 12
 $ regressors           : chr "data/genotype.bySample.thin"
 $ outcomes             : chr "data/expression.thin"
 $ ranef                : chr "data/ranef.tab"
 $ genespos             : chr "data/genespos.sqlite"
 $ snpspos              : chr "data/snpspos.sqlite"
 $ cis-window           : num 5e+05
 $ trans-pvalue         : num 1
 $ cis-pvalue           : num 1
 $ cores                : num 1
 $ max-regressions      : int 10000
 $ regressors-chunk-size: int 1000
 $ help                 : logi FALSE
reading outcomes data...
done:
'data.frame':	14 obs. of  3 variables:
 $ reporterID: chr  "1007_s_at" "1007_s_at" "1007_s_at" "1007_s_at" ...
 $ sampleID  : chr  "subj1-measure1" "subj1-measure2" "subj2-measure1" "subj2-measure2" ...
 $ expression: num  0.5 2.5 -4 -3 1.1 2.2 3.3 15 25 4 ...
reading random effects data...
done:
'data.frame':	7 obs. of  2 variables:
 $ sampleID: chr  "subj1-measure1" "subj1-measure2" "subj2-measure1" "subj2-measure2" ...
 $ R1      : Factor w/ 5 levels "subj1","subj2",..: 1 1 2 2 3 4 5
merging covariates data...
'data.frame':	7 obs. of  2 variables:
 $ sampleID: chr  "subj1-measure1" "subj1-measure2" "subj2-measure1" "subj2-measure2" ...
 $ R1      : Factor w/ 5 levels "subj1","subj2",..: 1 1 2 2 3 4 5
merging outcomes data...
'data.frame':	14 obs. of  4 variables:
 $ sampleID  : chr  "subj1-measure1" "subj1-measure2" "subj2-measure1" "subj2-measure2" ...
 $ reporterID: chr  "1007_s_at" "1007_s_at" "1007_s_at" "1007_s_at" ...
 $ expression: num  0.5 2.5 -4 -3 1.1 2.2 3.3 15 25 4 ...
 $ R1        : Factor w/ 5 levels "subj1","subj2",..: 1 1 2 2 3 4 5 1 1 2 ...
model formula:
expression ~ alleleCopyNumber + (1 | R1)

=> main loop started on Thu May  8 13:18:52 2014

no more lines in input: terminating...

=> main loop completed on Thu May  8 13:18:52 2014

2 regressors processed in 0.4180489 secs
 reading input: 0.003568649 secs
 joining data: 0.002083778 secs
 processing: 0.187505 secs
 filtering output: 0.0134244 secs
 writing output: 0.001532555 secs
 garbage collection: 0.1763299 secs
analysis completed.
